ajenti-core/aj/gate/stream.py

```diff
@@ -34,9 +35,10 @@ def __init__(self, obj, endpoint):
         self.endpoint = endpoint
 
     def serialize(self):
+        self.object_tmp = {k : (v.decode()  if isinstance(v, six.binary_type) else v) for k, v in self.object.items()}
```

Comment:
No need to put this into a property on self

Reply:
Also, why are we serializing everything into strings here, but into bytes in GateStreamResponse?

Reply:
Same problem with pickle in PY3 : the output will be a bytes, and I get a TypeError here https://github.com/ajenti/ajenti/blob/8f56ef4bec2119c232ba7e51494246299ddc7b81/ajenti-core/aj/gate/stream.py#L76
because "Object of type 'bytes' is not JSON serializable".

Reply:
Convert the whole dict is maybe not perfect, but I didn't find an easier way to do it.
Do you have a better idea ?

Reply:
As long as it works, I'm fine with it :)